mava/utils/make_env.py

```diff
@@ -91,10 +86,12 @@ def make_jumanji_env(
     # Create envs.
     env = jumanji.make(env_name, generator=generator, **config.env.kwargs)
     eval_env = jumanji.make(env_name, generator=generator, **config.env.kwargs)
-    env, eval_env = wrapper(env), wrapper(eval_env)
+    env, eval_env = wrapper(env, has_global_state=add_global_state), wrapper(
```

Comment:
Can we standardize the names also, I think I prefer `add_global_state` over `has_global_state`